includes/send_rx_functions.php

```diff
@@ -35,29 +36,30 @@ function send_rx_get_project_config($project_id, $project_type) {
     if (!in_array($project_type, array('patient', 'site'))) {
         return false;
     }
+    $externalModule = New ExternalModule();
 
-    $q = ExternalModules::getSettings('send_rx', $project_id);
-    if (!db_num_rows($q)) {
+    $projSettings = $externalModule->framework->getProjectSettings($project_id);
```

Comment:
```suggestion
    $projSettings = $externalModule->getProjectSettings($project_id);
```
Enabling the module fails if using `->framework`:

>REDCap crashed due to an unexpected PHP fatal error!
>Error message: Uncaught Error: Call to a member function getProjectSettings() on null in /var/www/html/modules/send_rx_v0.0.0/includes/send_rx_functions.php:40

Removing `->framework` makes it function properly. Likely that we don't have access to `framework` because the module does not specify a `framework_version`.